tests/unit/conftest.py

```diff
@@ -13,7 +13,13 @@ def user1_json():
             '"birthday": "2000-09-04", '\
             '"business_phones": ["+1 205 555 0108"], '\
             '"mobile_phone": null, '\
-            '"is_active": true}'
+            '"is_active": true, '\
+            '"additional_data": {"@odata.context": "https://graph.microsoft.com/v1.0/$metadata#users/$entity", '\
```

Comment:
Could we use json.loads to parse additional data?

Reply:
Hi @fey101, in order to minimize dependency on particular libraries, we have our own custom serialization/deserialization implementation for Kiota.